build/scripts/version-update/main.go

```diff
@@ -0,0 +1,138 @@
+package main
+
+import (
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+var releaseType string
+
+func init() {
+	flag.StringVar(&releaseType, "type", "", "Specify the release type ('before' or 'after')")
```

Comment:
Maybe `release-stage` instead of `type` (and change variables)? `releaseType` seems like it should be what type of release it is, vs what stage the release is in.

Reply:
Renamed type to release-stage and releaseType to releaseStage.